api/prometheus/v1/api.go

```diff
@@ -231,9 +231,9 @@ type API interface {
 	// Flags returns the flag values that Prometheus was launched with.
 	Flags(ctx context.Context) (FlagsResult, error)
 	// LabelNames returns all the unique label names present in the block in sorted order.
```

Comment:
I think this doc comment should be adjusted (it apparently didn't get adjusted when the time range was added, and now we have even matchers).

Reply:
Got it. I will update this.